.github/workflows/release.yml

```
@@ -151,3 +151,12 @@ jobs:
           draft: false
           prerelease: false
           generate_release_notes: true
+
+  deploy-docs:
+    needs:
+      - release-publish
+    uses: ./.github/workflows/deploy-docs.yml
+    with:
+      version: ${{ needs.gen-tag.outputs.tag }}
```

actionlint (1.7.7)

property "gen-tag" is not defined in object type {release-publish: {outputs: {}; result: string}}

(expression)